src/CSS/Nav.css

```diff
@@ -0,0 +1,10 @@
+.nav-container {
+  display: flex;
+}
+
+.nav-button {
+  margin: 25px 50px;
+  padding: 24px;
+  border-radius: 20px;
+  background-color: rgb(167, 164, 164);
```

Comment:
When you get to responsive design, don't forget about the text in the button needing to resize.